### ISSUE TITLE
add support for lstrip_blocks

### ISSUE THREAD

**Issue by RosanneZe** [Pylons/pyramid_jinja2#152]
Would it be possible to add support for the lstrip_blocks setting?

**Comment by stevepiercy** [comment]
Yes. Support for any Jinja2 configuration is possible. Pull requests to implement the feature request are welcome. See [requirements for contributions](https://pylonsproject.org/community-how-to-contribute.html) for full details.

Briefly that includes:

- your contribution
- tests with 100% coverage
- documentation of the feature
- signed [contributor agreement](https://github.com/Pylons/pyramid_jinja2/blob/master/CONTRIBUTORS.txt)

**Comment by RosanneZe** [comment]
I looked into it, and it is actually supported. It is not in the tests or the documentation, but it does work.

**Comment by stevepiercy** [comment]
Please do share you findings with a pull request.

**Comment by RosanneZe** [comment]
I didn't actually do any coding, just did a grep in the code. If you grep both `lstrip_blocks` and `trim_blocks`, you can see that they are both in the code, but only `trim_blocks` is in the tests. I thought it wasn't implemented, because it was not mentioned in the documentation, while `trim_blocks` was. (I checked that it worked by just adding it in my pyramid app and seeing if it made a difference).
